app/indexers/concerns/curator/indexer/attachment_indexer.rb

```diff
@@ -8,44 +9,42 @@ module AttachmentIndexer
         configure do
           each_record do |record, context|
             attachments = {}
+            attachment_fields = %i(byte_size content_type checksum).freeze
```

Comment:
I feel like `attachment_fields` and `has_one_attachment_types` could be constants rather than declared this way. Considering you're creating a new array with each iteration of the `each_record` loop.  Which eats into memory with each iteration until its garbage collected. Best to just declare it once as a constant.